docs/containers-transports.5.md

```diff
@@ -0,0 +1,53 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+# NAME
+containers-transports - description of supported transports for copying container images
+
+# DESCRIPTION
+
+**containers-storage:**_docker-reference_
+
+An image located in a local containers storage image store.
+Please refer to containers-storage.conf(5) for further information.
+
+**dir:**_path_
+
+An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files.
+This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+**docker://**_docker-reference_
+
+An image in a registry implementing the "Docker Registry HTTP API V2".
+By default, uses the authorization state in either `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(podman login)`.
```

Comment:
remove "either' as you don't have 'or' later in the sentence.  Your next sentence gets the point across just fine.

Reply:
Done.